### PR TITLE
Add helm requirements.yaml for quarks-job

### PR DESCRIPTION
**Pull request by edwardstudy**
[#171492037](https://www.pivotaltracker.com/story/show/171492037)

- Added helm requirements.yaml for quarks-job
- Bumped quarks-job to 491e84a

### CODE REVIEW

File: deploy/helm/cf-operator/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dependencies:
+  - name: quarks-job
+    version: 0.0.0+0.g06ad02b
```
Comment (manno) [inline deploy/helm/cf-operator/requirements.yaml]: I'd prefer it, if we had to change the version only in one file. Can we use `$QUARKS_JOB_HELM_VERSION` here?
Also this says 06ad02b and bin/include/dependencies says 491e84a

Comment (manno) [reply deploy/helm/cf-operator/requirements.yaml]: I missed the replace script

Comment (edwardstudy) [reply deploy/helm/cf-operator/requirements.yaml:3]: I used `0.0.0` as placeholder.

**Comment by edwardstudy** [comment]
Rebased and verified with helm v3.1.2

```
./bin/build-helm
Successfully packaged chart and saved it to: /Users/zidashaw/space/golang/workspace/src/github.com/cloudfoundry-incubator/cf-operator/helm/cf-operator-4.0.0+dirty-10.g5a91f8bf.tgz
```
image

**Pull request merged**